matching.go

```diff
@@ -209,6 +209,8 @@ func ReducePeople(people People, matcher external.Matcher, blacklist Blacklist,
 	mean, std := stat.MeanStdDev(componentsSize, nil)
 	if mean != mean {
 		mean = 0
+	}
+	if std != std {
```

Comment:
Does this have any advantage over `math.IsNan(std)`?

Reply:
https://github.com/golang/go/blob/master/src/math/bits.go#L39